src/foam/dao/RemoveChildrenOnRemoveDAO.js

```diff
@@ -0,0 +1,58 @@
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'RemoveChildrenOnRemoveDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  javaImports: [
+    'foam.mlang.MLang'
```

Comment:
Could you change this to 'static foam.mlang.MLang.*' and then avoid having to prefix MLangs with MLang?